FileBaseContext/Serializers/JsonRowDataSerializerFactory.cs

```diff
@@ -0,0 +1,12 @@
+﻿using Microsoft.EntityFrameworkCore.ChangeTracking;
+using Microsoft.EntityFrameworkCore.Metadata;
+
+namespace kDg.FileBaseContext.Serializers;
+
+public class JsonRowDataSerializerFactory : IRowDataSerializerFactory
```

Comment:
This type could optionally be `internal`

Reply:
fixed in 1a5c0c51b15ac1c2e0fd7f85123225f2f5956835